app/modules/wifi.c

```diff
@@ -506,8 +506,8 @@ static int wifi_station_get_ap_info4lua( lua_State* L )
     c_sprintf(debug_temp, " %-6d %-32s ", i, temp);
 #endif
 
-      memset(temp, 0, sizeof(temp));
-    if(strlen(config[i].password) >= 8)
+    memset(temp, 0, sizeof(temp));
+    if(strlen(config[i].password) >= 0) /* WPA = min 8, WEP = min 5 ASCII characters for a 40-bit key */
```

Comment:
Yeah, strlen will never be <0. Looking at this now after having cooled down from when I made the change, I think >0 is what is appropriate for this code that is copying the passwords only if they exist. The question becomes, should "pwd" be included the Lua table even if there is no password for the entry?